### ISSUE TITLE
When finished playing a playlist focus is lost.

### ISSUE THREAD

**Issue by jamielennox** [mazkolain-zz/spotimc#120]
I play a playlist and the now playing screen is showing. When it finishes i see no tracks loaded and nothing has focus and there is no way to navigate and choose something else.

**Comment by mazkolain-zz** [comment]
This is an ancient issue I forgot to track here. Thanks for reporting!

**Comment by nobodybody** [comment]
Hi!

I got the same problem.

Here's my xbmc log.
LOG 1: http://pastebin.com/tbGHae7a   (note: log gets deleted from pastebin after a month)
LOG 2: http://pastebin.com/rPnj2998   (note: log gets deleted from pastebin after a month)

LOG 1:
1) I started spotimc
2) Wen't into playlist --> then to check some settings --> back to playlist
3) Started playback of tune
4) Checked playback settings (Shuffle had "one dot", repeat had no dots - these doesn't matter, I've tried all)
5) Playback vizualisation comes on after a while (this doesn't matter either, tried actively keeping focus)
6) Tune ends, and spotimc says no tracks loaded even though there's a lot of tunes in the playlist. Spotimc also loses focus, and I have to restart xbmc.

LOG 2:
Same as first, but with some different playback settings. Repeat one dot, shuffle off. Same happens at the end.

Thanks!
